### ISSUE TITLE
System or SystemJS? (SystemJS is not defined)

### ISSUE THREAD

**Issue by chris-equis** [aurelia/beginner-kits#16]
I have downloaded the "beginners-kit" repository as a zip file, installed the dependencies with JSPM, installed http-server globally and started it, but in order to work I changed the below line from `index.html`

```javascript
// from
SystemJS.import('aurelia-bootstrapper'); // => throws ReferenceError: SystemJS is not defined
// to
System.import('aurelia-bootstrapper'); // => shows "Welcome to Aurelia!" heading on refresh
```

Did I miss something during the installation? I'm using `jspm 0.16.19`.

**Comment by EisenbergEffect** [comment]
Run jspm dl-loader --latest

**Comment by EisenbergEffect** [comment]
It's not related to jspm, but rather to the version of the loader that you are using. Somehow the kit must have got reverted to an earlier version of the loader. We will investigate. Thanks for letting us know.

**Comment by cecilphillip** [comment]
If you look through the source of systemjs on lines [4835](https://github.com/systemjs/systemjs/blob/master/dist/system.src.js#L4835) & [4855](https://github.com/systemjs/systemjs/blob/master/dist/system.src.js#L4855), you'll see System and SystemJS are aliases of each other

**Comment by chris-equis** [comment]
@cecilphillip 
It looks like they're aliases, but in reality with `SystemJS` it didn't work until I changed to `System`. In the beginning I thought that it might be a problem with jspm which brings systemjs but I have the last stable version installed. SystemJS version is `v0.19.9`.

Anyway, I don't find the switch from `SystemJS` to `System` a major issue since the fix is quite simple, if anyone else encounters this.

**Comment by EisenbergEffect** [comment]
Closing this issue. Current released beginner kits work correctly.

**Comment by myearick** [comment]
I appear to be having a very similar problem today.  I installed the kit and it ran fine the first time I ran it, but then I did a "jspm install underscore" and tried to run it again and I see the issue with SystemJS is not defined.  I tried to do the jspm dl-loader --latest, but that did not work.  The change from SystemJS to System does get further, but then I get an error loading the app "unexpected token <" (this seems to be a hack, but I thought I would try anyway).  Any suggestions would be appreciated.  This seems to be easily reproducible on my side (I have tried it numerous times).  

After installing underscore, I do see the following:

     Updating registry cache...
     Looking up npm:underscore
ok   Installed underscore as npm:underscore@^1.8.3 (1.8.3)
ok   Install tree has no forks.

warn core-js@2.1.0 is out of date. Use jspm dl-loader to update.
     Looking up loader files...
       es6-module-loader.js
       es6-module-loader.src.js
       es6-module-loader.js.map
       system.js
       system.src.js
       system.js.map

     Using loader versions:
       es6-module-loader@0.16.6
       systemjs@0.16.11
ok   Loader files downloaded successfully

ok   Install complete.

However, the jspm dl-loader --latest did not fix the error.
